scripts/install/install_unix.sh

```diff
@@ -13,165 +13,638 @@
-        if rpm -q observiq-otel-collector > /dev/null 2>&1; then
-            echo "Existing RPM installation detected, uninstalling..."
-            rpm -e "observiq-otel-collector"
-        else
-            echo "rpm was detected, but no observiq-otel-collector package is currently installed, skipping..."
-        fi
-    fi
+# This will display the results of an installation
+display_results()
+{
+    banner 'Information'
+    increase_indent
+    info "Collector Home:     $(fg_cyan "/opt/observiq-otel-collector")$(reset)"
+    info "Collector Config:   $(fg_cyan "/opt/observiq-otel-collector/config.yaml")$(reset)"
+    info "Start Command:      $(fg_cyan "sudo systemctl start observiq-otel-collector")$(reset)"
+    info "Stop Command:       $(fg_cyan "sudo systemctl stop observiq-otel-collector")$(reset)"
+    info "Logs Command:       $(fg_cyan "sudo journalctl -u observiq-otel-collector.service")$(reset)"
+    decrease_indent
+
+    banner 'Support'
+    increase_indent
+    info "For more information on configuring the collector, see the docs:"
+    increase_indent
+    info "$(fg_cyan "https://github.com/observiq/observiq-otel-collector/tree/main#observiq-opentelemetry-collector")$(reset)"
+    decrease_indent
+    info "If you have any other questions please contact us at $(fg_cyan support@observiq.com)$(reset)"
+    increase_indent
+    decrease_indent
+    decrease_indent
+
+    banner "$(fg_green Installation Complete!)"
+    return 0
 }
 
-main() {
-    # Determine system parameters
-    set_arch
-    set_package_type
-    set_file_name
+uninstall_package()
+{
+  case "$package_type" in
+    deb)
+      dpkg -r "observiq-otel-collector" > /dev/null 2>&1
+      ;;
+    rpm)
+      rpm -e "observiq-otel-collector" > /dev/null 2>&1
+      ;;
+    *)
+      error "Unrecognized package type"
+      return 1
+      ;;
+  esac
+  return 0
+}
+
+uninstall()
+{
+  observiq_banner
+
+  set_package_type
+  banner "Uninstalling observIQ OpenTelemetry Collector"
+  increase_indent
+
+  info "Checking permissions..."
+  root_check
+  succeeded
+
+  info "Stopping service..."
+  systemctl stop observiq-otel-collector > /dev/null || error_exit "$LINENO" "Failed to stop service"
+  succeeded
+
+  info "Disabling service..."
+  systemctl disable observiq-otel-collector > /dev/null 2>&1 || error_exit "$LINENO" "Failed to disable service"
+  succeeded
+
+  info "Removing package..."
+  uninstall_package || error_exit "$LINENO" "Failed to remove package"
+  succeeded
+  decrease_indent
+
+  banner "$(fg_green Uninstallation Complete!)"
+}
+
+main()
+{
+  if [ $# -ge 1 ]; then
+    while [ -n "$1" ]; do
+      case "$1" in
+        -v|--version)
+          version=$2 ; shift 2 ;;
+        -l|--url)
+          url=$2 ; shift 2 ;;
+        -x|--proxy)
+          proxy=$2 ; shift 2 ;;
+        -U|--proxy-user)
+          proxy_user=$2 ; shift 2 ;;
+        -P|--proxy-password)
+          proxy_password=$2 ; shift 2 ;;
+        -r|--uninstall)
```

Comment:
I notice we have `-l` for specifying the url, and `-r` for specifying uninstall. IMO, one of these should be `-u`

Reply:
Ok. I feel like it's confusing to new customers, is all. The only reason, from what I can see, that we made these flags weird, is because there was a `--service-user` option in previous scripts.